LLVM_FOR_RISCV.md

```diff
@@ -0,0 +1,123 @@
+
+# Building LLVM on Linux for RISC-V 64-bit Cross-Compilation
+
+This document outlines the steps to build LLVM on a Linux machine for cross-compiling C/C++ code for RISC-V 64-bit.
+
+## Before You Start
+
+Update your package manager to avoid errors during setup:
+
+```bash
+sudo apt-get update
+# or
+sudo apt update
+```
+
+## Setup RISC-V and LLVM Environments
+
+Install the required packages:
+
+```bash
+sudo apt-get -y install \
+  binutils build-essential libtool texinfo \
+  gzip zip unzip patchutils curl git \
+  make cmake ninja-build automake bison flex gperf \
+  grep sed gawk python3 bc \
+  zlib1g-dev libexpat1-dev libmpc-dev \
+  libglib2.0-dev libfdt-dev libpixman-1-dev
+```
+
+## Directory Setup for RISC-V and LLVM Files
+
+Create a directory for RISC-V files:
+
+```bash
+cd /data/users/$USER/condor # or your preferred workspace
+mkdir llvm
+cd llvm
+export PATH=/data/tools/bin:$PATH # adjust to your chosen install path
```

Comment:
`/data/tools/bin`

Reply:
can you make this path a local one instead of the shared dir? PATH=/data/tools/bin

Reply:
PATH=my/local/install/bin